algorithms/sort/bitonic_sort.py

```diff
@@ -0,0 +1,41 @@
+def bitonic_sort(arr, direction):
```

Comment:
I guess `direction` is the flag used to indicate the ascending or descending. I think this should be renamed to something like `reverse`, and work just like built-in `sort` method.

Also make the second parameter optional (by having a default value of sorting in increasing order). It should be callable by just `bitonic_sort([1, 3, 4, 2])`.

Reply:
Thank you for your comment,
I change my code to `direction` to `reverse` with default value True,